lib/rubocop/cop/variable_force/branch.rb

```diff
@@ -226,6 +226,21 @@ def always_run?
           end
         end
 
+        # case target
+        # in clause # in_clause
+        # else
+        #   else_body
+        # end
+        class CaseMatch < Base
+          define_predicate :target?,      child_index: 0
+          define_predicate :in_clause?,   child_index: 1..-2
+          define_predicate :else_body?,   child_index: -1
```

Comment:
The alignment space seems to be redundant.

```suggestion
          define_predicate :target?,     child_index: 0
          define_predicate :in_pattern?, child_index: 1..-2
          define_predicate :else_body?,  child_index: -1
```